interop-testing/src/main/java/io/grpc/testing/integration/AbstractInteropTest.java

```diff
@@ -337,7 +338,11 @@ public void tearDown() {
     stopServer();
   }
 
-  protected abstract ManagedChannel createChannel();
+  protected ManagedChannel createChannel() {
+    return createChannelBuilder().build();
+  }
+
+  protected abstract ManagedChannelBuilder<?> createChannelBuilder();
```

Comment:
This is an API breaking change. Users of `io.grpc:grpc-nterop-testing` implementing their own interop tests (testing custom transport or server implementation) are breaking.

Reply:
I would like make it non-abstract but throw unsupported exception.

Reply:
interop-testing has no API guarantees. As far as I'm concerned we only publish it so the command-line program is available.